piratebartender/js/app.js

```diff
@@ -0,0 +1,160 @@
+$(function() {
+
+
+    //ingredients constructor
+    var Ingredients = function(options) {
+        this.options = options;
+    };
+
+    var sweet = new Ingredients(['bunny ears', 'muddled peacock feathers', 'whipped cream']);
+
+
+    var salty = new Ingredients(['bacon', 'finely ground spider legs rim', 'olives']);
+
+
+    var spicy = new Ingredients(['essence of dart frog', 'Carolina Reaper peppers', 'dried remnants of a salsa jar']);
+
+
+    var sour = new Ingredients(['eyeballs', 'rotten apples', 'lizard tongues']);
+
+
+    var strong = new Ingredients(['bathtub gin', 'gasoline', 'liquid elephant tranquilizer']);
+
+
+    //pantry constructor: items to be added in case the thumbs down selected
+    var Pantry = function(stock) {
+        this.stock = stock;
+    }
+
+
+    var newPantry = new Pantry(['formaldehyde', 'arsenic', 'tiny colorful paper umbrella', 'dish water', 'wolfsbane', 'lemonade']);
```

Comment:
Just a personal preference of mine, I try to stay away from calling a variable something like `var newXyz`. The reason being `new` is one of many keywords used in JS and all it would take is a space and your are now saying `new Xyz` which is saying create a new instance of the Constructor Function `Xyz()`. You can see the other keywords here: [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar](url)